ack/readers/yandex_campaign/reader.py

```diff
@@ -36,7 +36,8 @@ def result_generator(self):
         api_client = ApiClient(self.token, YANDEX_DIRECT_API_BASE_URL)
         request_body = self._build_request_body()
         response = api_client.execute_request(url="campaigns", body=request_body, headers={})
-        yield response.json()
+        if response.json()["result"]:
+            yield response.json()
```

Comment:
Is there always a "result" key in the JSON?

Reply:
I checked in the API documentation and it seemed so. I can check again to be sure but it might be a good idea to add another check just in case. What do you think ?

Reply:
this is probably good enough but maybe you can add:
 ```python
if response.json().get("result", {}):
```
or something like that, in order to avoid an eventual error. I don't know the best practice about it but let me know what you think! 👍

Reply:
Ok thanks, indeed it may be better to use this condition instead. I'm also not sure if it is the best practice, as we might "hide" an issue instead of raising an error, but I think that in our use case it is ok to do so.